### PR TITLE
Contact card

### PR DESCRIPTION
**Pull request by davidpmccormick**
Relates to #4355 

Adding a `Contact` component which allows a team name, email and/or phone number to be displayed.

Screenshot 2019-07-02 at 14 25 58

Adds a `subtitle` field to the current `Team` (and we've changed the Prismic UI to display `Contact` instead of `Team`, to make the editorial flow make more sense).

~@jennpb Is the ability to add individuals a requirement? If so, I think this will require rethinking. Sticking with teams feels like it should be more future proof (even if individuals move roles, the teams remain the same), but not sure if it fits the needs.~

### CODE REVIEW

**Comment by jennpb** [comment]
Yes, unfortunately this isn't a straightforward team-for-team switch.

We do need individuals and roles, particularly for press releases where there is an assigned Media Manager, and where roles hold more cred.

**Comment by davidpmccormick** [comment]
It sounds like we'll need a new custom type in Prismic to allow the `Contact` component to reference _either_ a team _or_ an individual in a particular organisational role.

An e.g. `organisational-role` type could have a `title` (e.g. Lead Media Manager) and a `personInPost` which could possibly be a link to the `Person` type we already have, although it currently doesn't have `email` or `phone` fields – it's very much more geared towards editorial contributors, so do we need a different kind of `Person` type?

Ping @wellcometrust/experience-devs for thoughts.

File: prismic-model/js/teams.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import structuredText from './parts/structured-text';
+import text from './parts/text';
+
+const Teams = {
+  Contact: {
+    title: structuredText('Title', 'single', ['heading1']),
+    subtitle: text('Subtitle'),
```
Comment (jamesgorrie) [inline prismic-model/js/teams.js]: We're never going to want to put bits of markup in here?
Seems odd as the title allows it.

Comment (davidpmccormick) [reply prismic-model/js/teams.js]: Hmmmm… maybe I suppose. I came down on the side of it being YAGNI, but maybe that was too strong.

**Pull request merged**